### PR TITLE
Create folders according to the folder configuration

### PR DESCRIPTION
**Pull request by matthias-ronge**
When you create processes, subfolders are created in the task directory. Which, depended on the confused settings in the `kitodo_config.properties`. Creating the directory was done by a shell script, although that was not necessary. The configured `Folder`s and the existing switch _Create folder during the creation of a process_ did not have any effect.

Empty folders are now created during the process creation according to the `Folder` settings of the respective project. The folders are created without script, using the File Management.

### CODE REVIEW

**Comment by henning-gerhardt** [comment]
I'm voting against of removing the shell scripts as institute related changes inside are not possible any more. For example SLUB is using a partition separated folder structure to hold backup and restore times in good reasonable time and monitoring this partitions for used size is more efficient then using a multiple Terra Byte single partition. If this is get in then it is not possible to migrate for SLUB except later is a similar like now solution is found.

**Comment by Erikmitk** [comment]
I agree that we should discuss this internally before we take a next step. The file structure in our case (@SLUBDresden) is not flat but uses logic in the script to separate processes into partitions and to create symlinks accordingly. This mechanism would be completely gone and I don't know how we would get that back yet. The FileManagement-Module should do that part but I have no clue how it would be able to be configured and so on. The functionality has to move somewhere. So we have to talk about this first, @Kathrin-Huber.

**Comment by matthias-ronge** [comment]
data stores. Regardless of the question of whether even additionally an optional script hook might remain. The question is not only with SLUB, others have the same problem, yes, and if many clients in the future can work on one instance, it makes it more dramatic. The question is how to design such a function meaningfully. On the one hand, one could simply divide the set of processes; or all completed processes are moved to separate partition (there was such a function in the past, though broken) ... or how? What is the logic of SLUB?

File: Kitodo/src/main/java/org/kitodo/production/services/file/FileService.java [hunk 1]
```diff
@@ -695,6 +711,7 @@ public URI getProcessSubTypeURI(Integer processId, String processTitle, URI proc
 
         if (processDataDirectory == null) {
             try {
+                ProcessService processService = getProcessService();
```
[the remark this answers is not in the document]
Comment (matthias-ronge) [reply Kitodo/src/main/java/org/kitodo/production/services/file/FileService.java]: It's been dealt with.

**Comment by matthias-ronge** [comment]
I have rebuilt the branch so that the script will be used again.

**Comment by Kathrin-Huber** [comment]
please resolve conflicts

File: Kitodo/src/main/java/org/kitodo/production/services/data/ProcessService.java [hunk 2]
```diff
@@ -1620,6 +1610,7 @@ public void addToWikiField(User user, String message, Process process) {
     }
 
     /**
+<<<<<<< HEAD
```
Comment (Kathrin-Huber) [inline Kitodo/src/main/java/org/kitodo/production/services/data/ProcessService.java]: There is still merge-code

**Pull request merged**